management/azure_mgmt_compute/lib/azure_mgmt_compute.rb

```diff
@@ -2,4 +2,4 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 
-require 'generated/azure_mgmt_compute'
\ No newline at end of file
+require '2017-03-30/generated/azure_mgmt_compute'
```

Comment:
I forgot whether we decided to only include one api-version here or all of them.

Reply:
i'll see if I need to add other api versions, via running tests for another version too.

Reply:
from what I see in the below article, I think we may need all of them here.

>>>When packaging your gem files, you need to be careful of what is in your lib directory. Every gem you have installed gets its lib directory appended onto your $LOAD_PATH. This means any file on the top level of the lib directory could get required

Seems like that is how we expose our gems into $LOAD_PATH to be used by `require "xxx"`

http://guides.rubygems.org/patterns/

Reply:
yes, I think we will need them. I tried resources with tests for another api version and needed to add it, so I'll be adding it to others.